python/ray/util/sgd/tf/tf_runner.py

```diff
@@ -14,6 +15,7 @@ def _try_import_strategy():
     return tf.distribute.experimental.MultiWorkerMirroredStrategy
 
 
+@DeveloperAPI
```

Comment:
I don't think we need annotations for this right?

```
Developer APIs are lower-level methods explicitly exposed to advanced Ray
    users and library developers. Their interfaces may change across minor
    Ray releases.
```

`TFRunner` is purely internal and not even exposed to advanced users.